docker/README.md

```diff
@@ -0,0 +1,39 @@
+# Docker Information
+
+Developer Dockerfile is used to set up the development environment.
+It includes all the necessary tools, libraries, and dependencies required
+for development. This ensures that all developers work in a consistent
+environment, reducing the "it works on my machine" problem.
+Documentation Dockerfile is used to build and serve your project's documentation.
+
+Project uses Docker Compose to manage both the Developer Dockerfile and the
+Documentation Dockerfile. Docker Compose simplifies the process of building
+and running multiple Docker containers, ensuring that both the development
+environment and the documentation container are consistently set up and managed.
+
+## Build the docker image
+
+If you have docker compose installed, you can use this command
+
+    docker-compose -f docker/docker-compose.yaml up -d --build
+
+If you do not want to use docker compose, you can use this command instead
+
+    docker build -f docker/<dockerfile_name> -t <image_name> .
+
+This will build the image from the Dockerfile in the current directory (the `.`).
+
+## Create the docker container
+
+If you have docker compose installed, the easiest way is to call
+
+    docker exec -it <image_name> /bin/bash
```

Comment:
The given command is not using docker compose. The following command would be the one using docker compose:

    docker compose -f docker/docker-compose.yaml run <service_name>